hllplus/hllplus.go

```diff
@@ -191,17 +216,21 @@ func (s *HLL) Downgrade(precision, sparsePrecision uint8) error {
 	return nil
 }
 
+func (s *HLL) normalize() {
```

Comment:
This is the method that is not ready yet, at the moment it just switches from sparse to dense but it does not migrate existing/stored values, it's unfortunately a little tricky, but I will point you at the relevant code.